src/App.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Home from './pages/home';
+import Search from './pages/search';
+import './index.css';
 
 function App() {
   return (
     <div>
-      App Placeholder
+      <BrowserRouter>
```

Comment:
Seems like you use BrowserRouter twice. One time here and one time in `index.js`